code-challenges.test.js

```diff
@@ -1,69 +1,155 @@
 // ASSESSMENT 2: Coding Practical Questions with Jest
 
-// Please read all questions thoroughly
-// Pseudo coding is REQUIRED
-// If you get stuck, please leave comments to help us understand your thought process
+// const { throwStatement } = require("@babel/types")
 
-// Use test driven development to complete the following questions
-// Add appropriate dependencies: $ yarn add jest
+// // Please read all questions thoroughly
+// // Pseudo coding is REQUIRED
+// // If you get stuck, please leave comments to help us understand your thought process
 
-// Reminder: The test will call your function
-// Run the file with the following command: $ yarn jest
+// // Use test driven development to complete the following questions
+// // Add appropriate dependencies: $ yarn add jest
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multipled by 3.
+// // Reminder: The test will call your function
+// // Run the file with the following command: $ yarn jest
 
-// a) Create a test with expect statements for each of the variables provided.
+// // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiples by 3.
+
+// // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
+// // Expected output: [72, 81, 90, 99, 108]
 
 
 // b) Create the function that makes the test pass.
 
 
 
-// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
-// Expected output: "15 is divisible by three"
-const num2 = 0
-// Expected output: "0 is divisible by three"
-const num3 = -7
-// Expected output: "-7 is not divisible by three"
+// const num1 = 15
+// const num2 = 0
+// const num3 = -7
+// // Expected output: "15 is divisible by three"
+// // Expected output: "0 is divisible by three"
+// // Expected output: "-7 is not divisible by three"
+// // a describe method that lists the name of the function OR naming of the particular test.
 
+// describe("modThree", () => {
```

Comment:
Your test and functions should not be commented out. The test suite is built to hold all that information. And it makes it easier to review by knowing what code you want to showcase your work.